+++
title = "Hogwash: three methods for genome-wide association studies in bacteria"

date = "2020-11-18"
authors = ["Katie Saund", "Evan S. Snitkin"]
publication_types = ["2"]
publication = "Microbial Genomics"
publication_short = "Microbial Genomics"

abstract = "Bacterial genome-wide association studies (bGWAS) capture associations between genomic variation and phenotypic variation. Convergence based bGWAS methods identify genomic mutations that occur independently multiple times on the phylogenetic tree in the presence of phenotypic variation more often than is expected by chance. This work introduces hogwash, an open source R package that implements three algorithms for convergence based bGWAS. Hogwash additionally contains two burden testing approaches to perform gene- or pathway-analysis to improve power and increase convergence detection for related but weakly penetrant genotypes. To identify optimal use cases, we applied hogwash to data simulated with a variety of phylogenetic signals and convergence distributions. These simulated data are publicly available and contain the relevant metadata regarding convergence and phylogenetic signal for each phenotype and genotype. Hogwash is available for download from GitHub."
image_preview = ""
selected = false
projects = []

url_pdf = "files/2020_hogwash_methods_published_version.pdf"
url_preprint = ""
url_code = "https://github.com/katiesaund/hogwash"
url_dataset = ""
url_project = "https://github.com/katiesaund/simulate_data_for_convergence_based_bGWAS"
url_slides = ""
url_video = ""
url_poster = ""
url_source = "https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000469"
links = []

math = false
highlight = true
+++
